api/models/User.js

```diff
@@ -161,6 +161,15 @@ module.exports = bookshelf.Model.extend(merge({
     .digest('hex')
   },
 
+  reactivate: function () {
+    return this.set({ active: true })
```

Comment:
I thought that .set didn't actually do a .save()? We usually do .save({ active: true })